### DISCUSSION TITLE
issue after migrated

### DISCUSSION THREAD

**Discussion by dev060** [WalletConnect/walletconnect-monorepo#2878] [Developer Support]
I migrated walletconnect to 2.0 
It works fine on local but popup is not showing on live.
this issue is showing on console.
![Screen Shot 2023-06-30 at 00 49 26](https://github.com/WalletConnect/walletconnect-monorepo/assets/91896655/45661074-b107-4fa3-92a0-4970eb912dfe)

**Comment by chris13524** [comment]
Hey @dev060! Thanks for reporting this. However, I'm not sure it is related to WalletConnect.

To help investigate, can you please share:
- A minimal reproducible example that demonstrates the issue
- Versions of WalletConnect SDKs you are using

**Comment by dev060** [comment]
"@walletconnect/ethereum-provider": "^2.8.6",
"@walletconnect/modal": "^2.5.9",

I am using these packages

below is the code which is used when connect using walletconnect (svelte/rollup)

			const provider = await EthereumProvider.init({
				projectId: 'my projectId', // required
				chains: [10], // required
				showQrModal: true // requires @walletconnect/modal
			})
			await provider.enable();

**Comment by dev060** [comment]
![image_2023-06-30_01-00-36](https://github.com/WalletConnect/walletconnect-monorepo/assets/91896655/61be63ba-a3c0-44b4-964e-59f0155aa572)
 here is another screenshot of the issue

**Reply by chris13524** [reply]
cc @ganchoradkov @0xAsimetriq

**Comment by dev060** [comment]
![Screen Shot 2023-06-30 at 13 17 56](https://github.com/WalletConnect/walletconnect-monorepo/assets/91896655/97cd67df-f67f-4c4a-a298-68d049993def)

PLUS: I see this error when running, deploying the app.

**Comment by dev060** [comment]
@chris13524 
Is there any update? This is pretty urgent since our protocol is live. Many users are not able to use mobile.

**Reply by glitch-txs** [reply]
Could you share a minimal reproduction of the issue in a repo or codesandbox?

**Comment by dev060** [comment]
https://github.com/dev060/WalletConnect2Sample

this is the sample app to reproduce the issue.

Please check if you can see above 2 issues

second issue(fromMiliSeconds) is stopping rendering always on develop mode

**Comment by dev060** [comment]
Note: first issue is disappeared in production mode.

but build is always failed in the server (timeout error - more than 15 mins)

**Comment by dev060** [comment]
Finally I found the reason of the first issue. it's crashed with coinbase wallet sdk.

...
"@coinbase/wallet-sdk": "^3.7.1",

How to fix it?

@chris13524 @glitch-txs

**Comment by chris13524** [comment]
I tested your example and ran `npm install` and `npm run dev` and was able to successfully connect my wallet. I don't see the errors that you are seeing.

You can ignore the verify.walletconnect.com error as this can be fixed by enabling Verify API in your cloud dashboard.

<img width="1094" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/1979423/21195ee7-9bb5-4b7d-848b-a619e5ffc3dd">

Unfortunately we will need a better example to help further. Please minimize the number of dependencies and list clear reproduction steps :)

**Comment by dev060** [comment]
Thanks for your reply, first issue is happened when use coinbase sdk together.
and second issue should be happened whenever you update something while running with npm run dev.
So I need to run npm run dev again whenever I change something.
You can see this second issue in demo app I shared

Thanks

**Comment by dev060** [comment]
@chris13524 
Any update on the second issue? it's pretty urgent.

Thanks

**Comment by finessevanes** [comment]
@dev060 the `No Matching Keys` is a benign error. I am closing this since we are already aware of this issue. Please follow this [issue](https://github.com/WalletConnect/walletconnect-monorepo/issues/2326) for the latest update. For any other issues, please open a separate discussion. Thank you!

**Reply by dev060** [reply]
Hi @finessevanes 
"No Maching Keys" issue is not affecting anything. I didn't mean that issue. even I didn't report this issue.
You are confusing something.
I am talking about this issue. 
https://user-images.githubusercontent.com/91896655/250001748-97cd67df-f67f-4c4a-a298-68d049993def.png
this issue is pretty annoying and blocking our development.
You can reproduce this issue in the sample app I shared.
New change is not reflected on the app because of that issue. Need to run "npm run dev" again to see new change.
Imagine running "npm run dev" again whenever make any change.
I hope you to take a look this issue asap.
Thanks

**Comment by boidushya** [comment]
@dev060 Following up on your second issue, we were unable to replicate this on our [end](https://codesandbox.io/p/github/boidushya/WalletConnect2Sample/main?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522cljz8zpct000g3b7arsfmh0id%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522cljz8zpct000b3b7a3ry8pk7d%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522cljz8zpct000f3b7a5y7shgsk%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522cljz8zpct000d3b7a9r98l5mz%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522cljz8zpct000b3b7a3ry8pk7d%2522%253A%257B%2522id%2522%253A%2522cljz8zpct000b3b7a3ry8pk7d%2522%252C%2522activeTabId%2522%253A%2522cljz8zpct000a3b7alzbn0f66%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cljz8zpct000a3b7alzbn0f66%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252FREADME.md%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%252C%257B%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252F.codesandbox%252Ftasks.json%2522%252C%2522id%2522%253A%2522cljz93cpa01ia3b7a5ptycdaj%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%257D%252C%2522cljz8zpct000d3b7a9r98l5mz%2522%253A%257B%2522id%2522%253A%2522cljz8zpct000d3b7a9r98l5mz%2522%252C%2522activeTabId%2522%253A%2522cljz932cx00wo3b7atpwxnccm%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522port%2522%253A5000%252C%2522id%2522%253A%2522cljz932cx00wo3b7atpwxnccm%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%252F%2522%257D%255D%257D%252C%2522cljz8zpct000f3b7a5y7shgsk%2522%253A%257B%2522id%2522%253A%2522cljz8zpct000f3b7a5y7shgsk%2522%252C%2522activeTabId%2522%253A%2522cljz92qfl00jf3b7a8o3aut3k%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cljz8zpct000e3b7av0hoy2fx%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TERMINAL%2522%252C%2522shellId%2522%253A%2522cljz8zppy000gfndd9ujcf2go%2522%257D%252C%257B%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522id%2522%253A%2522cljz92qfl00jf3b7a8o3aut3k%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%255D%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)
<img width="2560" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/26746725/cfb68dc6-b26b-4710-977b-848952184e1c">
This may be caused by a broken lockfile, have you tried removing yarn.lock and node_modules and trying again?

**Reply by dev060** [reply]
Yes,
Did you change something and save? It's not rendering because of the second issue I mentioned.
Yes it runs fine first time but if you make change and save, second issue happens.

Thanks for your help

**Reply by dev060** [reply]
![Screen Shot 2023-07-13 at 00 44 41](https://github.com/WalletConnect/walletconnect-monorepo/assets/91896655/73b2a597-4ca1-403a-a0de-e0472bafc2db)
Run the app with "npm run dev"
And try to update text from "Hello World" to "Hello user" and save(ctrl+s).
then You can see second issue in terminal.

**Reply by boidushya** [reply]
I see it now, is this happening on any other framework or just svelte?

**Reply by dev060** [reply]
I didn't check on other framework.

**Reply by dev060** [reply]
Note. 
different issues are happened whenever save changes

**Comment by dev060** [comment]
@boidushya Is there any luck?
